R/utilities-mpi-error-handling.R

```diff
@@ -7,6 +7,14 @@ checkSimulationLoaded <- function(simulation, simulationFilePath) {
   return(identical(simulation$sourceFile, simulationFilePath))
 }
 
+#' @title checkPopulationLoaded
+#' @description Check that the population has been loaded on the core.
+#' @param population the population object to be checked if loaded successfully
+#' @export
+checkPopulationLoaded <- function(population) {
+  return(identical(class(population)[1], "Population"))
```

Comment:
What about you guys copy the method ValidateIsOfType from OSPSuite-R
This can be used generically in a much more natural way I believe 
In fact, this type of check should be implemented in any public function

Reply:
Done.  We have this method in reportingengine, should there only be one version?